crates/re_types/source_hash.txt

```diff
@@ -1,4 +1,4 @@
 # This is a sha256 hash for all direct and indirect dependencies of this crate's build script.
 # It can be safely removed at anytime to force the build script to run again.
 # Check out build.rs to see how it's computed.
-9db9060d979f575ae784945806bfb6916cb8a5ea12e8edcdd1d29ed79b668cfd
```

Comment:
This file shouldn't be in the repo O.o

Reply:
Ha. I knew I had a weird feeling about this -- Im not entirely sure how this came back to life. I think it was after I ran some pixi task, hmmm...